assets/js/components/adminbar/LegacyAdminBarWidgets.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Legacy admin bar widgets.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import AnalyticsAdminbarWidgetOverview from '../../modules/analytics/components/adminbar/AnalyticsAdminbarWidgetOverview';
+import SearchConsoleAdminbarWidgetOverview from '../../modules/search-console/components/adminbar/SearchConsoleAdminbarWidgetOverview';
+import { Row } from '../../material-components';
+import AdminBarZeroData from './AdminBarZeroData';
+import AnalyticsInactiveCTA from '../AnalyticsInactiveCTA';
+
+export default function LegacyAdminBarWidgets() {
+	const [ zeroDataSearchConsole, setZeroDataSearchConsole ] = useState( false );
+	const [ zeroDataAnalytics, setZeroDataAnalytics ] = useState( false );
+	const [ zeroData, setZeroData ] = useState( false );
+
+	useEffect( () => {
+		if ( zeroDataSearchConsole && zeroDataAnalytics ) {
```

Comment:
Related to my comment above, we only need to consider the Analytics "zero data" state if Analytics is actually active and connected. If there is no data for the active modules, we need to show the `AdminBarZeroData` state, even though in theory there might be data after activating another module.

See here how this currently behaves incorrectly (bottom part of the screenshot is the legacy one):
![Screen Shot 2021-01-14 at 2 25 12 PM](https://user-images.githubusercontent.com/3531426/104658309-5ad5f380-5677-11eb-9f6f-e232d3192f37.png)

(this screenshot also shows the weird font sizes referenced in my other comment)

Reply:
I've resolved this now also and I've updated the new admin bar to only show a single gathering data component in this case to match the legacy design. At the same time, I've removed the plain divs which we were using to style the admin bar in favour of using the MUI `<Cell />` components.

Reply:
@benbowler Felix is pointing out how the top (new) implementation is correctly showing the zero data component here but this (legacy) one is not.

In the current version of this branch, both new and old do not handle this correctly:

![image](https://user-images.githubusercontent.com/1621608/106457265-63139880-6497-11eb-87a6-52ad9f202984.png)

If SC is zero data and Analytics is not connected, it should show the single zero data component.